File: studyOC/studyOC/Scheme和Universal/LinkViewController.m
//
//  LinkViewController.m
//  studyOC
//
//  Created by Hahn on 2021/9/15.
//

#import "LinkViewController.h"
#import "LinkModel.h"

@interface LinkViewController ()

@end

@implementation LinkViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.view.backgroundColor = [UIColor blackColor];
}

- (void)setModel:(LinkModel *)model
{
    
    UILabel *url = [[UILabel alloc] initWithFrame:CGRectMake(0, 80, 250, 80)];
    [self.view addSubview:url];
    
    UILabel *para = [[UILabel alloc] initWithFrame:CGRectMake(0, 160, 250, 500)];
    [self.view addSubview:para];
    
    url.text = model.url;
    para.text = model.para.description;
    para.numberOfLines = 0;
}

@end

File: studySwift/studySwift/Scheme和Universal/LinkViewController.swift
//
//  LinkViewController.swift
//  studySwift
//
//  Created by Hahn on 2021/9/16.
//

import UIKit

class LinkViewController: UIViewController {

    public var model:LinkModel?
    

    override func viewDidLoad() {
        super.viewDidLoad()
        
        
        let label = UILabel(frame: CGRect(x: 0, y: 80, width: 200, height: 80))
        label.text = model?.url
        self.view.addSubview(label)
        
        let label1 = UILabel(frame: CGRect(x: 0, y: 180, width: 200, height: 80))
        label1.numberOfLines = 0
        label1.text = "id =  \(model?.para?.id ?? "") +  name =  + \(model?.para?.name ?? "") +  age =  + \(model?.para?.age ?? "")"
        self.view.addSubview(label1)
        
    }
    
    

}
